tensorflow/g3doc/tutorials/mnist/pros/index.md

```diff
@@ -409,8 +410,14 @@ for i in range(20000):
     print("step %d, training accuracy %g"%(i, train_accuracy))
   train_step.run(feed_dict={x: batch[0], y_: batch[1], keep_prob: 0.5})
 
-print("test accuracy %g"%accuracy.eval(feed_dict={
-    x: mnist.test.images, y_: mnist.test.labels, keep_prob: 1.0}))
+acc = 0 
+num_batch = mnist.test.num_examples/50
+for i in range(num_batch):
+  batch = mnist.test.next_batch(50)
+  acc += accuracy.eval(feed_dict={x: batch[0], y_: batch[1], keep_prob: 1.0})
+
+acc /= num_batch
+print("test accuracy %g"%acc)
```

Comment:
This code is for mnist tutorial (with 10000 test examples) and not for general process. Also, to process unbatched test data, one needs >4GB gpu memory. This modification is for relieving gpu memory usage.